lib/cli.js

```diff
@@ -473,11 +496,22 @@ Promise.resolve()
     if (!linted.output) {
       return;
     }
+
     process.stdout.write(linted.output);
+
     if (linted.errored) {
       process.exitCode = 2;
+    } else if (maxWarnings !== undefined && linted.maxWarningsExceeded) {
+      const foundWarnings = linted.maxWarningsExceeded.foundWarnings;
+
+      process.stdout.write(
+        `Stylelint found too many warnings:${EOL}` +
```

Comment:
Can we tweak the message to make it consistent [with how other errors (e.g. "Invalid Option" ones)](https://cloud.githubusercontent.com/assets/808227/14970067/063e5802-10be-11e6-8774-6557d80c443b.png) are formatted, please? e.g. :

```js
  process.stdout.write(
        chalk.red(`Max warnings exceeded: `) +
          `${foundWarnings} found. ` +
          chalk.dim(`${maxWarnings} allowed${EOL}${EOL}`)
      );
```

Reply:
@jeddy3 Looks good, updated.